installer/PowerToysSetup/Product.wxs

```diff
@@ -356,29 +357,33 @@
-        <File Source="$(var.BinX64Dir)modules\shortcut_guide.dll" KeyPath="yes" />
-      </Component>
+    <DirectoryRef Id="FancyZonesInstallFolder" FileSource="$(var.BinX64Dir)modules\">
       <Component Id="Module_FancyZones" Guid="C6B5272E-6ED4-4B80-B0E7-2FF0355D8CF4" Win64="yes">
         <File Source="$(var.BinX64Dir)modules\fancyzones.dll" KeyPath="yes" />
-        <File Source="$(var.BinX64Dir)modules\FancyZonesEditor.exe" >
+        <File Source="$(var.BinX64Dir)modules\FancyZones\FancyZonesEditor.exe" >
           <netfx:NativeImage Id="FancyZonesEditor.exe" Platform="64bit" Priority="0" />
         </File>
-        <File Source="$(var.BinX64Dir)modules\ControlzEx.dll" />
-        <File Source="$(var.BinX64Dir)modules\MahApps.Metro.dll" />
-        <File Source="$(var.BinX64Dir)modules\Microsoft.Xaml.Behaviors.dll" />
-        <File Source="$(var.BinX64Dir)modules\FancyZonesEditor.exe.config" />
-        <File Source="$(var.BinX64Dir)modules\Microsoft.Bcl.AsyncInterfaces.dll" />
-        <File Source="$(var.BinX64Dir)modules\System.Buffers.dll" />
-        <File Source="$(var.BinX64Dir)modules\System.Memory.dll" />
-        <File Source="$(var.BinX64Dir)modules\System.Numerics.Vectors.dll" />
-        <File Source="$(var.BinX64Dir)modules\System.Runtime.CompilerServices.Unsafe.dll" />
-        <File Source="$(var.BinX64Dir)modules\System.Text.Encodings.Web.dll" />
-        <File Source="$(var.BinX64Dir)modules\System.Text.Json.dll" />
-        <File Source="$(var.BinX64Dir)modules\System.Threading.Tasks.Extensions.dll" />
-        <File Source="$(var.BinX64Dir)modules\System.ValueTuple.dll" />
+        <File Source="$(var.BinX64Dir)modules\FancyZones\ControlzEx.dll" />
+        <File Source="$(var.BinX64Dir)modules\FancyZones\MahApps.Metro.dll" />
+        <File Source="$(var.BinX64Dir)modules\FancyZones\Microsoft.Xaml.Behaviors.dll" />
+        <File Source="$(var.BinX64Dir)modules\FancyZones\FancyZonesEditor.exe.config" />
+        <File Source="$(var.BinX64Dir)modules\FancyZones\Microsoft.Bcl.AsyncInterfaces.dll" />
+        <File Source="$(var.BinX64Dir)modules\FancyZones\System.Buffers.dll" />
+        <File Source="$(var.BinX64Dir)modules\FancyZones\System.Memory.dll" />
+        <File Source="$(var.BinX64Dir)modules\FancyZones\System.Numerics.Vectors.dll" />
+        <File Source="$(var.BinX64Dir)modules\FancyZones\System.Runtime.CompilerServices.Unsafe.dll" />
+        <File Source="$(var.BinX64Dir)modules\FancyZones\System.Text.Encodings.Web.dll" />
+        <File Source="$(var.BinX64Dir)modules\FancyZones\System.Text.Json.dll" />
+        <File Source="$(var.BinX64Dir)modules\FancyZones\System.Threading.Tasks.Extensions.dll" />
+        <File Source="$(var.BinX64Dir)modules\FancyZones\System.ValueTuple.dll" />
       </Component>
+    </DirectoryRef>
+
+    <DirectoryRef Id="ModulesInstallFolder" FileSource="$(var.BinX64Dir)modules\">
```

Comment:
Why not to move Module_FancyZones component here?

Reply:
that will be done as a separate PR